common/sonar.js

```diff
@@ -11,11 +11,20 @@ function sonarStatusReferesher() {
 
     var rnd = Math.round(Math.random() * 1000000000000);
     var marker = pointURI.indexOf("&rnd=");
+    var wasPointRefreshedEarlier = marker !== -1
```

Comment:
Actually, this variable is kind of excessive here, but I've added it to make the code more self-documented.